code/modules/mod/modules/modules_general.dm

```diff
@@ -16,6 +16,27 @@
 	var/max_items = 7
 	var/obj/item/storage/backpack/modstorage/bag
 
+/obj/item/mod/module/storage/serialize()
+	var/data = ..()
+	var/list/content_list = list()
+	data["content"] = content_list
+	for(var/old_bag in bag.contents)
+		var/atom/movable/AM = old_bag
+		// Copied from storage serialization.
+		content_list.len++
+		content_list[content_list.len] = AM.serialize()
+	return data
```

Comment:
Why not just serialize the bag?

Reply:
Because the bag will have different variables that are set by the module's init and both the mod and the suit will hold a ref to it. It was just less risky to copy the contents and let the module handle the bag itself.